.circleci/config.yml

```diff
@@ -285,27 +285,27 @@ jobs:
 
               echo "Build official Docker image"
               cp ~/repo/.docker/Dockerfile .
```

Comment:
I'm aware that there are some files which should not be contributed but somehow resulted from our fork. I'll remove them once we finalized the requirement on our fork.